### PR TITLE
feat(refund): allow collective admins to issue refunds

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

File: server/graphql/v1/mutations/orders.js [hunk 1]
```diff
@@ -845,7 +845,7 @@ export async function refundTransaction(_, args, req) {
   //   b. Host Collective receiving the donation
   //   c. Site Admin
```
Comment (znarf) [inline server/graphql/v1/mutations/orders.js]: @xdamman can you perhaps change these comments which are now outdated?

Comment (xdamman) [reply server/graphql/v1/mutations/orders.js]: Ah sorry, only seeing your comment after merging. Shall I do another PR or just push another commit for that small change in master?

Comment (xdamman) [reply server/graphql/v1/mutations/orders.js]: New PR: https://github.com/opencollective/opencollective-api/pull/2662

**Comment by Betree** [comment]
Would have been great to make a security review of the refund process before merging that.

**Pull request merged**